mp/src/game/client/hl2/hud_credits.cpp

```diff
@@ -137,8 +120,50 @@ class CHudCredits : public CHudElement, public Panel
     float m_flCreditsPixelHeight;
 
     HFont m_hTFont;
+
+    int m_iMusicGUID;
+
+    ConVarRef sv_unlockedchapters;
 };
 
+using namespace vgui;
+
+static CHudCredits *g_pCreditsDialog = nullptr;
+
+//-----------------------------------------------------------------------------
+// Purpose: Constructor
+//-----------------------------------------------------------------------------
+CHudCredits::CHudCredits() : BaseClass(nullptr, "HudCredits"), sv_unlockedchapters("sv_unlockedchapters")
+{
+    g_pCreditsDialog = this;
+
+    SetParent(enginevgui->GetPanel(PANEL_GAMEUIDLL));
+    SetProportional(true);
+
+    // for mouse speedup
+    SetMouseInputEnabled(true);
+    SetKeyBoardInputEnabled(true);
+
+    SetMoveable(false);
+    SetSizeable(false);
+    SetDeleteSelfOnClose(false);
+    SetMinimizeToSysTrayButtonVisible(false);
+    SetSysMenu(nullptr);
+
+    SetTitleBarVisible(false);
+    SetMenuButtonVisible(false);
+    SetMenuButtonResponsive(false);
+    SetMinimizeButtonVisible(false);
+    SetMaximizeButtonVisible(false);
+    SetCloseButtonVisible(false);
+
+    int iWidth, iTall;
+    surface()->GetScreenSize(iWidth, iTall);
+    SetBounds(0, 0, iWidth, iTall);
+
+    sv_unlockedchapters.SetValue("15");
```

Comment:
Since [this is set to 99 by default](https://github.com/momentum-mod/game/blob/develop/mp/src/game/server/gameinterface.cpp#L2004) we can remove this ref, as it's old behavior from HL2 I believe.